target_iceberg/target.py

```diff
@@ -73,6 +73,21 @@ class TargetIceberg(Target):
                         "Otherwise new data will be appended.",
             default="",
         ),
+        th.Property(
+            "upsert_data_for_streams",
+            th.StringType,
```

Comment:
Another option is to use ArrayType, which validates if the user is setting an array without we needing to do anything.